style.css

```diff
@@ -135,11 +144,226 @@ body {
+ }
+
+  .main__title {
+    margin: 0 0 33px 6px;
+    padding-top: 25px;
+    font: bold 30px/39px "pt_serif", arial, helvetica, sans-serif;
+    text-align: center;
+ }
+
+  .main__subtitle {
+    margin: 0 25px;
+    padding-bottom: 25px;
+    font: 500 16px/19px "Roboto", arial, helvetica, sans-serif;
+    text-align: center;
+ }
+
+  .main__button {
+    padding: 14px 22px;
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+ }
+
+                        /* Бургер меню */
+
+  #menu__toggle {
+    opacity: 0;
+ }
+
+  .menu__btn {
+    display: flex;
+    align-items: center;
+    position: fixed;
+    left: 326px;
+    top: 37px;
+    width: 25px;
+    height: 3px;
+    cursor: pointer;
+    z-index: 1;
+ }
+
+  .menu__btn > span,
+  .menu__btn > span::before,
+  .menu__btn > span::after {
+    display: block;
+    position: absolute;
+    width: 100%;
+    height: 3px;
+    background-color: #0056ec;
+ }
+
+  .menu__btn > span::before {
+    content: '';
+    top: -8px;
+ }
+
+  .menu__btn > span::after {
+    content: '';
+    top: 8px;
+ }
+
+  .menu__box {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    width: 375px;
+    height: 100%;
+    margin: 0;
+    padding: 86px 0;
+    text-align: center;
+    background-color: #fff;
+ }
+
+  .menu__item {
+    display: block;
+    font-family: "Roboto", arial, helvetica, sans-serif;
+    font-size: 20px;
+    font-weight: 500;
+    top: 79px;
+    padding: 15px;
+    margin: 10px;
+    font-size: 29px;
+ }
+
+  .menu__item:hover {
+    background-color: #cfd8dc;
+ }
+
+  #menu__toggle:checked ~ .menu__btn > span {
+    transform: rotate(45deg);
+ }
+
+  #menu__toggle:checked ~ .menu__btn > span::before {
+    top: 0;
+    transform: rotate(0);
+ }
+
+  #menu__toggle:checked ~ .menu__btn > span::after {
+    top: 0;
+    transform: rotate(90deg);
+ }
+
+  #menu__toggle:checked ~ .menu__box {
+    visibility: visible;
+    left: 0;
+ }
+}
+
+                         /* Экран 1024px */
+
+  @media (min-width: 769px) and (max-width: 1024px) {
+   .wrapper {
+     width: 91%;
+ }
+
+   .header {
+     margin-bottom: 66px;
+ }
+
+  .menu__item {
+    font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+ }
+
+   .main {
+     padding: 35px 50% 126px 0;
+     background: top right no-repeat url(img/mainImg1.svg);
+     background-position-y: 20px;
+  }
+
+   .main__title {
+     margin: 0 0 39px 0;
+     font: bold 50px/66px "pt_serif", arial, helvetica, sans-serif;
+ }
+
+   .main__subtitle {
+     margin: 0 0 35px 10px;
+     font: 500 17px/22px "Roboto", arial, helvetica, sans-serif;
+     max-width: 350px;
+}
+
+   .main__button {
+     font: 500 14px/16px "Roboto", arial, helvetica, sans-serif;
+     padding: 16px 20px;
+ }
+}
+
+                      /* Экран 1366px */
+
+  @media (min-width: 1025px) and (max-width: 1366px) {
+   .wrapper {
+    width: 68%;
+  }
+
+   .header {
+     margin-bottom: 66px;
+  }
+
+   .menu__item {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6297664)